### PR TITLE
HBASE-25630 Set switch compaction after bulkload default as false

### PR DESCRIPTION
**Pull request by nyl3532016**

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 27s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 25s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 42s |  master passed  |
| +1 :green_heart: |  compile  |   8m 11s |  master passed  |
| +1 :green_heart: |  checkstyle  |   2m  0s |  master passed  |
| +0 :ok: |  refguide  |   3m 29s |  branch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect.  |
| -1 :x: |  spotbugs  |   9m  3s |  root in master has 1 extant spotbugs warnings.  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 14s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 26s |  the patch passed  |
| +1 :green_heart: |  compile  |   8m  7s |  the patch passed  |
| -0 :warning: |  javac  |   8m  7s |  root generated 1 new + 1561 unchanged - 1 fixed = 1562 total (was 1562)  |
| +1 :green_heart: |  checkstyle  |   2m  1s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  2s |  The patch has no ill-formed XML file.  |
| +0 :ok: |  refguide  |   3m 22s |  patch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect.  |
| +1 :green_heart: |  hadoopcheck  |  17m 26s |  Patch does not cause any errors with Hadoop 3.1.2 3.2.1 3.3.0.  |
| +1 :green_heart: |  spotbugs  |   0m 52s |  hbase-common in the patch passed.  |
| +1 :green_heart: |  spotbugs  |   2m  9s |  hbase-server in the patch passed.  |
| +1 :green_heart: |  spotbugs  |   9m  2s |  root generated 0 new + 0 unchanged - 1 fixed = 0 total (was 1)  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 41s |  The patch does not generate ASF License warnings.  |
|  |   |  85m 18s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/3022 |
| Optional Tests | dupname asflicense javac refguide xml spotbugs hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 62cebc167057 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 830d2895b2 |
| Default Java | AdoptOpenJDK-1.8.0_282-b08 |
| refguide | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/1/artifact/yetus-general-check/output/branch-site/book.html |
| spotbugs | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/1/artifact/yetus-general-check/output/branch-spotbugs-root-warnings.html |
| javac | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/1/artifact/yetus-general-check/output/diff-compile-javac-root.txt |
| refguide | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/1/artifact/yetus-general-check/output/patch-site/book.html |
| Max. process+thread count | 141 (vs. ulimit of 30000) |
| modules | C: hbase-common hbase-server . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/1/console |
| versions | git=2.17.1 maven=3.6.3 spotbugs=3.1.12 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 27s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 26s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m 20s |  master passed  |
| +1 :green_heart: |  compile  |   2m 50s |  master passed  |
| +1 :green_heart: |  shadedjars  |   6m 49s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   3m 43s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 13s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 58s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 51s |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 51s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   6m 50s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   3m 42s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 171m 34s |  root in the patch passed.  |
|  |   | 210m 32s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/1/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/3022 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux b3f3774714c1 4.15.0-136-generic #140-Ubuntu SMP Thu Jan 28 05:20:47 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 830d2895b2 |
| Default Java | AdoptOpenJDK-11.0.10+9 |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/1/testReport/ |
| Max. process+thread count | 6294 (vs. ulimit of 30000) |
| modules | C: hbase-common hbase-server . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/1/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

File: src/main/asciidoc/_chapters/hbase-default.adoc [hunk 1]
```diff
@@ -763,9 +763,10 @@ Maximum retries. This is a maximum number of iterations
 Request Compaction after bulkload immediately.
 If bulkload is continuous, the triggered compactions may increase load,
 bring about performance side effect.
+(During HBase-2.0, "hbase.compaction.after.bulkload.enable" default value is true)
```
Comment (Apache9) [inline src/main/asciidoc/_chapters/hbase-default.adoc:766]: 'For HBase-2.x, the default value for "hbase.compaction.after.bulkload.enable" is true'.

Comment (nyl3532016) [reply src/main/asciidoc/_chapters/hbase-default.adoc]: yes, I amend it

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  7s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 25s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m  3s |  master passed  |
| +1 :green_heart: |  compile  |   8m 37s |  master passed  |
| +1 :green_heart: |  checkstyle  |   2m 16s |  master passed  |
| +0 :ok: |  refguide  |   3m 43s |  branch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect.  |
| -1 :x: |  spotbugs  |  10m  3s |  root in master has 1 extant spotbugs warnings.  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 12s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 53s |  the patch passed  |
| +1 :green_heart: |  compile  |   8m 29s |  the patch passed  |
| +1 :green_heart: |  javac  |   8m 29s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   2m 19s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  2s |  The patch has no ill-formed XML file.  |
| +0 :ok: |  refguide  |   3m 36s |  patch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect.  |
| +1 :green_heart: |  hadoopcheck  |  19m  6s |  Patch does not cause any errors with Hadoop 3.1.2 3.2.1 3.3.0.  |
| +1 :green_heart: |  spotbugs  |   0m 52s |  hbase-common in the patch passed.  |
| +1 :green_heart: |  spotbugs  |   2m 19s |  hbase-server in the patch passed.  |
| +1 :green_heart: |  spotbugs  |  10m 18s |  root generated 0 new + 0 unchanged - 1 fixed = 0 total (was 1)  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 36s |  The patch does not generate ASF License warnings.  |
|  |   |  93m 23s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/2/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/3022 |
| Optional Tests | dupname asflicense javac refguide xml spotbugs hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 4a091d5f5308 4.15.0-136-generic #140-Ubuntu SMP Thu Jan 28 05:20:47 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 190c253c4e |
| Default Java | AdoptOpenJDK-1.8.0_282-b08 |
| refguide | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/2/artifact/yetus-general-check/output/branch-site/book.html |
| spotbugs | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/2/artifact/yetus-general-check/output/branch-spotbugs-root-warnings.html |
| refguide | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/2/artifact/yetus-general-check/output/patch-site/book.html |
| Max. process+thread count | 126 (vs. ulimit of 30000) |
| modules | C: hbase-common hbase-server . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/2/console |
| versions | git=2.17.1 maven=3.6.3 spotbugs=3.1.12 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 27s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 28s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m  2s |  master passed  |
| +1 :green_heart: |  compile  |   2m 47s |  master passed  |
| +1 :green_heart: |  shadedjars  |   6m 40s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   3m 43s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 12s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   4m  5s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 50s |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 50s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   6m 41s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   3m 35s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 168m 57s |  root in the patch passed.  |
|  |   | 207m 37s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/2/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/3022 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux d29c1fb8dd08 4.15.0-136-generic #140-Ubuntu SMP Thu Jan 28 05:20:47 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 190c253c4e |
| Default Java | AdoptOpenJDK-11.0.10+9 |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/2/testReport/ |
| Max. process+thread count | 6408 (vs. ulimit of 30000) |
| modules | C: hbase-common hbase-server . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/2/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 29s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  2s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 24s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 41s |  master passed  |
| +1 :green_heart: |  compile  |   2m 20s |  master passed  |
| +1 :green_heart: |  shadedjars  |   6m 34s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 59s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 15s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 25s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 23s |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 23s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   6m 37s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 57s |  the patch passed  |
||| _ Other Tests _ |
| -1 :x: |  unit  | 336m 32s |  root in the patch failed.  |
|  |   | 371m 15s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/2/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/3022 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux f0ad48b7ef0c 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 190c253c4e |
| Default Java | AdoptOpenJDK-1.8.0_282-b08 |
| unit | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/2/artifact/yetus-jdk8-hadoop3-check/output/patch-unit-root.txt |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/2/testReport/ |
| Max. process+thread count | 6018 (vs. ulimit of 30000) |
| modules | C: hbase-common hbase-server . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3022/2/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Review by Apache9 — APPROVED** [review]

**Pull request merged**